### ISSUE TITLE
Non-finite location and/or size for viewport

### ISSUE THREAD

**Issue by robsalasco** [anthonynorth/snapbox#9]
Hi! I'm running snapbox under debian buster using a rmarkdown file and render() command. I don't have issues on macos but adding the layer to ggspatial cause the infamous "non-finite location and/or size for viewport" error.

Hope you can find a solution, thank you guys!

```   
layer_mapbox(st_bbox(isocrona), 
             mapbox_api_access_token = "",
             map_style = mapbox_light(), 
             retina = F, scale_ratio = 0.5) + 
```

```
Error in grid.Call.graphics(C_setviewport, vp, TRUE) : 
  non-finite location and/or size for viewport
```

**Comment by robsalasco** [comment]
update: I tried to build under R 4.0.2 using the rocker geospatial image and it gets stuck trying to render the map :/

**Comment by MilesMcBain** [comment]
I haven't been able to reproduce this on Ubuntu 18.04 :grimacing:

**Comment by MilesMcBain** [comment]
@robsalasco since you say "infamous", do you have a link to some info about this error in mind?

**Comment by robsalasco** [comment]
@MilesMcBain yeah it's very common and it have [multiple reasons](https://www.google.com/search?q=Non-finite+location+and/or+size+for+viewport+site:stackoverflow.com&client=safari&rls=en&sa=X&ved=2ahUKEwiXrNnZ3LzrAhU1C9QKHWBaCIQQrQIoBHoECFcQBQ&biw=1242&bih=712&dpr=2) . Are you using a docker image? I'm trying to use the library on rocker geospatial I tried 3.6.1 and 4.0.2. In the first the non-finite error appeared and in the 4.0.2 version snapbox doesn't work because it gets stuck doing something. I use the library inside a rmarkdown file ;).
